security/integration/utils/kubernetes.go

```diff
@@ -155,6 +155,10 @@ func CreatePod(clientset kubernetes.Interface, namespace string, image string, n
 	return clientset.CoreV1().Pods(namespace).Get(name, metav1.GetOptions{})
 }
 
+func DeleteSecrets(clientset kubernetes.Interface, namespace string, name string) error {
```

Comment:
nit: this should be `DeleteSecret` since it only takes a single secret each time.

Reply:
Renamed